emperor/support_files/js/sceneplotview3d.js

```diff
@@ -338,20 +349,44 @@ define([
 
     // shortcut to the index of the visible dimension and the range object
     var x = this.visibleDimensions[0], y = this.visibleDimensions[1],
-        z = this.visibleDimensions[2], range = this.dimensionRanges;
+        z = this.visibleDimensions[2], range = this.dimensionRanges,
+        is2D = z === null, axesPadding = 1.07;
```

Comment:
1.07?

Reply:
Would you mind adding some text/comment about that selection and if possible how it was selected? 1.07 is pretty specific.